### PR TITLE
Create the logo component

### PR DESCRIPTION
**Pull request by tborres**
- create the logo component 
- allow the branding name to display or hide
- update favicon to use the coral color rather than filtered

Closes [B2IO-1](https://b2io.atlassian.net/browse/B2IO-1?atlOrigin=eyJpIjoiZDNkYTE4Nzg5YTVhNGJjOWJhOGM4NzRkMDRjNjkxZDUiLCJwIjoiaiJ9)

### CODE REVIEW

File: src/components/Logo.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+export const Logo: FC<LogoProps> = ({
+  height = 'auto',
+  showName = false,
+  ...props
+}) => {
+  const width = !showName || height === 'auto' ? height : height * (212 / 59);
+
+  return (
```
Comment (FrankyMartz) [inline src/components/Logo.tsx:15]: There's a visual difference between the Logo w/"Base Two" text and the Logo only.

We would have to ask Bonfire why both were constructed uniquely; however, my initial assumption is that perhaps these disparities are to improve visual perception at different scales. Until we get that clarity, we should probably keep those two separate for the time being.

> Context: While any vector image can scale from favicon to billboard; it's always best to keep the SVG at or near the dimensions it was designed for. For instance, a favicon is better suited with a logo w/broad shapes whereas a figure image benefits from granular details.

**BaseTwo Logo w/Text**
b2_logo_text


**BaseTwo Logo Only**

b2_logo

File: src/components/Layout.tsx [hunk 2]
```diff
@@ -25,6 +26,7 @@ export const Layout: FC<LayoutProps> = ({ children, title = 'Base Two' }) => {
         <meta name="viewport" content="initial-scale=1.0, width=device-width" />
       </Head>
       <header>
+        <Logo height={30} />
```
Comment (FrankyMartz) [inline src/components/Layout.tsx:29]: When working with images that dynamically scale, I've found it best to control the dimensions of an image indirectly.

> Context: In some situations where scale requires either `width` or `height` in half pixel; the engine does a better job at making this decision.

```jsx
<div style="width:30px;"><Logo width="100%" height="auto" /><div>
```

**Review by FrankyMartz — COMMENTED** [review]
Overall solid start. Outside of concerns commented inline; I also recommend adding a `NOTE:` that the `node_modules` directory may need to be deleted if the developer was working with yarn previously.

As I understand it, **YarnPkg** and **NPM** do not share package reconciliation, so utilization is not interchangeable.

File: README.md [hunk 3]
```diff
@@ -39,41 +39,35 @@ You'll need to set up Git and NPM before you can run this project locally. The s
...
-   npm install --global yarn
-   ```
-
 </details>
 
 **Install Packages**
 
 ```
-λ yarn
+λ npm i
```
[the remark this answers is not in the document]
Comment (k8hansen) [reply README.md:47]: Same question here.

**Review by k8hansen — APPROVED** [review]
LGTM!

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (tborres) [reply README.md:47]: FYI: I moved the yarn to npm change to a different PR: #516

**Review by sludemann — APPROVED** [review]
🎉

**Pull request merged**